.github/workflows/bump_plugins.yml

```
@@ -0,0 +1,109 @@
+name: Update plugins
+on:
+  workflow_dispatch:
+
+jobs:
+  # ugly way to share the plugin list
+  plugin_list:
+    runs-on: ubuntu-latest
+    outputs:
+      plugins: ${{ steps.set-list.outputs.plugins }}
+    steps:
+    - name: Write the plugin list
+      run: echo '["foreman_ansible"]' > plugin_list.json
+    - name: Set the plugins list
+      id: set-list
+      run: |
+        PLUGINS=$(echo $(cat plugin_list.json))
```

Why the echo? Isn't this sufficient?
```suggestion
        PLUGINS=$(cat plugin_list.json)
```